code/datums/status_effects/debuffs.dm

```diff
@@ -390,6 +390,26 @@
 			I.take_damage(100)
 	return ..()
 
+// This status effect holds no behavior on its own. This is intentional.
+// Its strength is checked by /mob/living/get_confusion().
+
+/// A status effect used for specifying confusion on a living mob.
+/// Checked by /mob/living/get_confusion().
+/// For most cases, you should be able to just modify the living's confused variable directly.
+/// This status effect exists mostly so you can quickly nullify the effect of a confusion.
+/datum/status_effect/confusion
+	id = "confusion"
+	status_type = STATUS_EFFECT_MULTIPLE
+	tick_interval = -1
+	alert_type = null
+
+	/// The strength of the confusion. Will eventually lower by 1 per status effect tick.
+	var/strength = 0
+
+/datum/status_effect/confusion/on_creation(mob/_, _strength = 0)
```

Comment:
`mob/_` ?

Reply:
Unused variable.

Reply:
give unused variable proper names in the future, as they may get some kind of use

Reply:
When someone wants to use the variable, they should just rename it then, no? Underscores as unused variables are pretty common elsewhere.

Reply:
Not really a pattern we use here. Even if we did use this pattern we'd go with it prefixing an informative name, not just keep it empty. Give it one and don't introduce this standard on a language that doesn't check whether the variable is used or not.

You seem to be debating every single request, even inconsequential ones like this one. At this time I'm spending more time arguing with you than reviewing your code. I don't mind clarifying things, but the endless back-and-forth is a bit much.

Reply:
I've been feeling that too, I apologize if I'm coming off as confrontational. I just want to narrow things down and make sure we're both on the same page here as to my intentions with the code.

Reply:
All good. To be fair the status effect one was mostly you clarifying what you meant.